VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.scss

```diff
@@ -0,0 +1,94 @@
+@import '~govuk-frontend/govuk/all';
+
+.panel-container {
+  background: govuk-colour('black');
+  color: govuk-colour('white') !important;
+  width: 300px;
+  min-height: 100px;
+  max-height: 50%;
+  padding-right: 10px;
+  position: fixed;
+  z-index: 100;
+  -webkit-box-shadow: -3px 0px 5px 0px rgba(0, 0, 0, 0.2);
+  box-shadow: -3px 0px 5px 0px rgba(0, 0, 0, 0.2);
+  left: 0;
+  vertical-align: top;
+  opacity: 0.7;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+
+.table-br {
+  border-color: govuk-colour('dark-grey') !important;
+}
+
+.panel-icon {
+  height: 25px;
+  margin: 8px 3px;
+}
+
+.cell-icon {
+  width: 31px;
+  display: inline-block;
+}
+
+.icon-op:hover {
+  opacity: 1;
+}
+
+.icon-op {
+  opacity: 0.5;
+}
+
+.vh-name {
+  display: inline-block;
+  width: 210px;
+}
+
+.vh-twidth {
+  width: 100px;
+}
+
+.grey-btn {
```

Comment:
Is govuk secondary button not suitable here?

Reply:
no, Neil has it different, I  followed to his prototype